pkgs/desktops/xfce/applications/xfmpc/default.nix

```diff
@@ -0,0 +1,58 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+, autoreconfHook
+, xfce4-dev-tools
+, libxfce4ui
+, libxfce4util
+, pkg-config
+, vala
+, gtk3
+, glib
+, libmpd
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "xfmpc";
+  # Last release is too old
+  version = "0.3.1-2024-05-29";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.xfce.org";
+    owner = "apps";
+    repo = "xfmpc";
+    rev = "cf40dffec6e9b80abb1f1aa6d7dceef4790173dc";
+    hash = "sha256-moCWSLGBJuWM4/lRJi6D3w38iJeCntLo3Vl/eVfu7lw=";
+  };
+  nativeBuildInputs = [
+    autoreconfHook
+    xfce4-dev-tools
```

Comment:
In `xdtAutogenPhase` I would [expect](https://gitlab.xfce.org/xfce/xfce4-dev-tools/-/blob/4bd037dbe567d9161e7a65108e31f955194a570b/scripts/xdt-autogen.in#L437) xdt-autogen to trigger autoreconf, do we have to keep `autoreconfHook` here triggering autoreconf twice?

Reply:
> In `xdtAutogenPhase` I would [expect](https://gitlab.xfce.org/xfce/xfce4-dev-tools/-/blob/4bd037dbe567d9161e7a65108e31f955194a570b/scripts/xdt-autogen.in#L437) xdt-autogen to trigger autoreconf, do we have to keep `autoreconfHook` here triggering autoreconf twice?

You were right! Ended up using only `xdtAutogenPhase`.